docs/Account/methods.md

```diff
@@ -35,8 +36,8 @@ _required params are explicitly mentioned_
   - biconomyPaymasterApiKey(`string`) OR paymaster (`IPaymaster`): one can either pass paymaster API key or custom paymaster instance to use the paymaster.
   - entryPointAddress (`string`): DEFAULT_ENTRY_POINT_ADDRESS will be used if not passed, otherwise the passed address will be used. On specific chains like Chiliz Mainnet it is a different address, so will need to be passed explicitly. Refer to below notes on this.
   - activeValidationModule (`BaseValidationModule`): The run-time validation module (must be one of enabled validation modules) to sign and validate next userOp.
-  - rpcUrl (`string`): RPC URL of the chain
-  - index (`number`): index to create multiple smart accounts for an EOA
+  - rpcUrl (`string`): This is an optional param for viem and ethers, It's advised to pass RPC url in case of custom signers such as privy, dynamic etc. If rpcUrl is not provided then a default public rpc will be used - which will likely be heavily throttled and can often silently fail.
```

Comment:
This is phrased a bit strangely, as it sounds like the param is optionally for viem and ethers, but the opposite is true.  It's essentially mandatory for signers that are *not* viem and ethers. Might be more readable to say:

`Recommended for signers that are not a viem wallet or an ethers signer`